addons/website_payment/static/src/js/website_payment_form.js

```diff
@@ -118,6 +119,10 @@ checkoutForm.include({
         const targetId = ev.target.id;
         if (targetId.indexOf("amount") >= 0) {
             this.txContext.amount = ev.target.value;
+            if (targetId === "other_amount_value") {
+                //we need to do this because the custom amount is represented by two input
+                this.$('input[id="other_amount"]').val(ev.target.value);
```

Comment:
Let's try to write it without JQuery so that we don't have to remove it during forward port.